snap/snapcraft.yaml

```diff
@@ -0,0 +1,122 @@
+#
+# To build:
+#
+# Using multipass, one should enable a large multipass VM using:
+#
+#   SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=8G snapcraft
+#
+# Alternatively, one can build in a lxd container using:
+#
+#   snapcraft --use-lxd
+#
+# Install using something like:
+#
+#   sudo snap install --devmode bpftrace_0.9.4-20200312-1403-11814f2_amd64.snap
+#
+name: bpftrace
+summary: high-level tracing language for Linux eBPF
+description: BPFtrace is a high-level tracing language for
+  Linux enhanced Berkeley Packet Filter available in recent
+  Linux kernels. BPFtrace uses LLVM as a backend to compile scripts
+  to BPF-bytecode and makes use of BCC for interacting with the Linux
+  BPF system, as well as existing Linux tracing capabilities like
+  kernel dynamic tracing, user-level dynamic tracing and tracepoints.
+
+confinement: strict
+grade: stable
+base: core18
+architectures: [amd64]
+assumes: [snapd2.37]
+adopt-info: bpftrace
+
+parts:
+    bcclibs3:
+        source: https://github.com/iovisor/bcc.git
+        source-tag: v0.8.0
```

Comment:
is there a specific reason this is still 0.8.0? Or can we safely upgrade it to 0.12 later?

Reply:
I used 0.8.0 as this was the version I had verfied to word for a build using the versions of the support libraries used in the snap. I suspect it can be updated to 0.12 or later if it can be verified to work fine.